test/parallel/test-cluster-shared-handle-bind-privileged-port.js

```diff
@@ -21,6 +21,7 @@
 
 'use strict';
 const common = require('../common');
+const { readFileSync } = require('fs');
```

Comment:
Can you move this to inside of the try/catch. There is no need to load `fs` if OS is not linux.